bcgov/HOUS-permit-portal
# Feature/bphh 571/coord lookup rebase

## Description

Adds ability to lookup addresses based on pids, ability to find things based on pins

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-515
https://hous-bssb.atlassian.net/browse/BPHH-1545

## Steps to QA

Backend for developers to test new functions against LTSA and Geocoder Wrapper.
On the select of a new PIN, type 10 Semana Crescent.  It should give you an address, you select it, it has no PID.  You can try a PIN, it will try to validate it.  It will show if it cannot verify it.  If you delete the PIN it should clear out.

@derekyau - please review the new ruby gems, they require local installs / binaries for geo location.  (open the gem documentation)
